crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -105,6 +105,27 @@ impl GlobalTransform {
         self
     }
 
+    #[doc(hidden)]
```

Comment:
Why add doc(hidden) here?

Reply:
I added it because the other *creation* methods (looking_at and from_*) are all doc(hidden).

I think it's a bit strange that they are hidden but users don't usually create GlobalTransforms.

Reply:
hmm yeah then i guess we can just leave this as-is for now and then circle back if/when we need to with docs.